src/domUpdates.js

```diff
@@ -1,80 +1,212 @@
-      console.log(domUpdates.randomNumber)
-      greetUser(data.wcUsersData[domUpdates.randomNumber -1])
-       return data.wcUsersData;
-        })
-    .catch(err => {
-        console.log(err);
-        alert('Sorry, the user information failed to load. Try again later.');
-    })
   },
-
-    grabIngredients() {
-    return fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/ingredients/ingredientsData')
-    .then(response => {
-        return response.json()
-    }).then(ingredientData => {
-        return ingredientData.ingredientsData;
-        })
-    .catch(err => {
-        console.log(err);
-        alert('Sorry, the ingredients failed to load. Try again later.');    
-  })
- },
-mergeFetchTimelines() {
-   Promise.all([domUpdates.grabUsers(), domUpdates.grabIngredients()])
-   .then(values => {
-     let usersData = values[0]
-     let ingredientsData = values[1]
-     let currentUser =domUpdates.createUser(usersData)
-     console.log(currentUser)
-     let pantry = domUpdates.createPantry(currentUser,ingredientsData)
-     console.log(pantry)
-        displayPantry(pantry)
-   })
-    .catch(err => {
-        console.log(err);
-        alert('Sorry, the information failed to load. Try again later.');
-    })
+  
+   greetUser(user) {
+    const userName = document.querySelector('.user-name');
+    if(userName === null){
+      return 
+    }
+    userName.innerText += ' ' + user.name.split(' ')[0] + ' ' + user.name.split(' ')[1][0];
   },
-  createUser(usersData) {
-    let currentUser = usersData.find(user => {
-        let parsedID = parseInt(user.id);
-        console.log(domUpdates.randomNumber)
-        return parsedID === domUpdates.randomNumber})
-        return currentUser
+  
+   displayPantry(pantry) {
+    if(pantryArea === null) {
+      return 
+    }
+    pantry = pantry.contents
+    pantryArea.innerHTML = ''
+    pantry.forEach(ingInPantry => {
+      pantryArea.innerHTML += `<article class="single-ingredient-card">
+    <article class="ingredient-name-area">
+      <h2 class="ingredient-name">${ingInPantry.name}</h2>
+    </article>
+    <article class="ingredient-quantity-section">
+      <button>-</button>
+      <p class = 'ingredient-quantity'>${ingInPantry.amount}</p>
+      <button>+</button>
+    </article>
+  </article>`
+    });
   },
-  createPantry(currentUser,ingredientsData){
-  currentUser.pantry.forEach(pantryItem => {
-  let currentIngredient = ingredientsData.find(ingredient => {
-      return pantryItem.ingredient === ingredient.id;
-  })
-  pantryItem.name = currentIngredient.name;
-  pantryItem.estimatedCostInCents = currentIngredient.estimatedCostInCents;
-  })
-  //what if we made a class here?
-  return currentUser.pantry
+     displayFavorites(favorites) {
+       if(favorites.length === 0) {
+         return
+       }
+      recipeCards.innerHTML = ''
+      favorites.forEach(recipe =>{
+        recipeCards.innerHTML += `<article id = ${recipe.id} class="single-recipe-card ">
```

Comment:
Nice, I like the 'return' here if no favorites exist. I'm assuming this essentially means that the button wont do anything, which might be better than showing the user a blank screen